packages/blocks/table/src/Table.tsx

```diff
@@ -426,7 +424,7 @@ export const Table: BlockComponent<AppProps> = ({
     setHiddenColumns(newHiddenColumns);
 
     // @todo throttle this call
-    updateRemoteColumns({ hiddenColumns: newHiddenColumns });
+    updateRemoteColumnsRef.current({ hiddenColumns: newHiddenColumns });
```

Comment:
This change was made to make the usage of `updateRemoteColumnsRef` consistent across the block.